test/e2e/steps/modifyTask.step.js

```diff
@@ -0,0 +1,47 @@
+const TaskAsserts = require('../asserts/Task.assert');
+const Context = require('../../data/Context');
+const LoginPage = require('../pages/LoginPage');
+const { Given, When, Then } = require('cucumber');
+const TaskPage = require('../pages/TaskPage');
+const DashboardPage = require('../pages/DashboardPage');
+
+Given(/^user is in home page$/, function () {
+  LoginPage.open(Context.page.endpoints.base);
+  DashboardPage.listViewHeader.waitForExist();
+});
+
+When(/^user creates a task$/, function (dataTable) {
+  let task = dataTable.rowsHash();
+  DashboardPage.floatingMenuButton.click();
+  TaskPage.setFloatingName(task.name);
+  TaskPage.setFloatingDescription(task.description);
+  TaskPage.setPriorityButton.click();
+  TaskPage.priorityOption(task.priority.toLowerCase()).click();
+  TaskPage.floatingStatusButton.click();
+  TaskPage.floatingTaskStatus(task.status.toLowerCase()).click();
+  TaskPage.createTaskButton.click();
+});
+
+When(/^user modifies "([^"]*)" fields$/, function (taskName, dataTable) {
+  let task = dataTable.rowsHash();
+  TaskPage.getTaskInList(taskName).waitForExist(3000);
+  TaskPage.getTaskInList(taskName).click();
+  let elm = $('.task-name__overlay').getText().length;
+  TaskPage.modalName.click();
+  for(let i = 0; i < elm; i++) {
+    browser.keys('\uE003');
+  }
+  TaskPage.setModalName(task.name);
+  TaskPage.modalDescription.click();
+  TaskPage.setModalDescription(task.description);
+  TaskPage.closeTaskModal.click();
+});
+
+Then(/^"([^"]*)" values should be$/, function (taskName, dataTable) {
+  let task = dataTable.rowsHash();
+  TaskPage.getTaskInList(taskName).waitForExist(3000);
+  TaskPage.getTaskInList(taskName).click();
+  TaskAsserts.assertName(task.name);
+  TaskAsserts.assertDescription(task.description);
+  TaskPage.closeTaskModal.click();
```

Comment:
Is it necessary to close this?

Reply:
Yes, otherwise the modal will block the necessary element for the deleteList hook